<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prescription_00</class>
 <widget class="QWizardPage" name="Prescription_00">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>570</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="patientBox">
     <property name="title">
      <string>Patient</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lastLabel">
        <property name="text">
         <string notr="true">LASTNAME</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="firstLabel">
        <property name="text">
         <string notr="true">FIRSTNAME</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="medicationLabel">
       <property name="text">
        <string>MEDICATION/BARCODE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="medicationField">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEDICATION</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LOT NUMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FORM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STRENGTH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UNIT SIZE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STOCK</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>EXPIRATION</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>medicationField</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>medicationField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
